.eslintcache

```diff
@@ -0,0 +1 @@
+{"/Users/diamond/Documents/semantic-math/math-rules/lib/transforms.js":{"size":603,"mtime":1493574423000,"hashOfConfig":"39a17q","results":{"filePath":"/Users/diamond/Documents/semantic-math/math-rules/lib/transforms.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/diamond/Documents/semantic-math/math-rules/lib/rules.js":{"size":2897,"mtime":1493599472000,"hashOfConfig":"39a17q","results":{"filePath":"/Users/diamond/Documents/semantic-math/math-rules/lib/rules.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/diamond/Documents/semantic-math/math-rules/test/matcher_test.js":{"size":7454,"mtime":1493599527000,"hashOfConfig":"39a17q","results":{"filePath":"/Users/diamond/Documents/semantic-math/math-rules/test/matcher_test.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/diamond/Documents/semantic-math/math-rules/test/rules_test.js":{"size":6146,"mtime":1493599543000,"hashOfConfig":"39a17q","results":{"filePath":"/Users/diamond/Documents/semantic-math/math-rules/test/rules_test.js","messages":[],"errorCount":0,"warningCount":0}}}
```

Comment:
We probably shouldn't be commit this file.  Can you add it to .gitignore and remove it from the diff?